index.html

```diff
@@ -174,6 +174,13 @@ <h2 class="text-center text-uppercase text-white">Team</h2>
           <div class="col-lg-3 mr-auto">
             <p class="lead">Mark Conrad has a wide range of experience in research and blockchain including working as a Data Scientist at lifeID, a company building a Blockchain solution for digital identity, as well as leading the University of Washington Blockchain Society as President of the Bothell chapter.</p>
           </div>
+
+          <div class="col-lg-3 ml-auto">
+            <p class="lead">Prateek Adhikaree has a Masters in Information Systems from Northeastern University. He has full-stack software engineering experience. He worked as a Blockchain Engineer for almost 2 years where he built on multiple blockchain networks as part of BaaS (Blockchain as a Service) offering. He enjoys solving problems and is excited to contribute to cryptocurrencies being used for its intended purpose.</p>
+          </div>
+          <div class="col-lg-3 mr-auto">
```

Comment:
I think this was supposed to me ml-auto (text is showing on the wrong side)... I will fix later though :)

Reply:
Changing it to`ml-auto` puts a lot of extra space between the bio and the image.